packages/core/src/browser/diff-uris.ts

```diff
@@ -26,7 +26,7 @@ export namespace DiffUris {
 
     export function decode(uri: URI): URI[] {
         if (uri.scheme !== DIFF_SCHEME) {
-            throw (`The URI must have scheme "diff". The URI was: ${uri}.`);
+            throw Error(`The URI must have scheme "diff". The URI was: ${uri}.`);
```

Comment:
`new` is missing?

Reply:
Hmm it seems to work either way:

```
> throw new Error("allo")
Error: allo
    at repl:1:7
    at ContextifyScript.Script.runInThisContext (vm.js:50:33)
    at REPLServer.defaultEval (repl.js:239:29)
    at bound (domain.js:301:14)
    at REPLServer.runBound [as eval] (domain.js:314:12)
    at REPLServer.onLine (repl.js:440:10)
    at emitOne (events.js:120:20)
    at REPLServer.emit (events.js:210:7)
    at REPLServer.Interface._onLine (readline.js:282:10)
    at REPLServer.Interface._line (readline.js:631:8)
> throw Error("allo")
Error: allo
    at repl:1:7
    at ContextifyScript.Script.runInThisContext (vm.js:50:33)
    at REPLServer.defaultEval (repl.js:239:29)
    at bound (domain.js:301:14)
    at REPLServer.runBound [as eval] (domain.js:314:12)
    at REPLServer.onLine (repl.js:440:10)
    at emitOne (events.js:120:20)
    at REPLServer.emit (events.js:210:7)
    at REPLServer.Interface._onLine (readline.js:282:10)
    at REPLServer.Interface._line (readline.js:631:8)
```

But I'll add it for consistency (I fixed the others using the auto-fixed and it added "new", so it seems to be the preferred way).

Reply:
New version pushed.